alpha-build-machinery/make/examples/multiple-binaries/Makefile

```diff
@@ -1,3 +1,22 @@
 include $(addprefix ../../, \
 	golang.mk \
 )
+
+CROSS_BUILD_BINDIR :=_bin
```

Comment:
elsewhere we call this `output`. `_bin` is super ugly.

Reply:
the only intent I had was to save a subdir otherwise you'd need `_output/bin/...`. I don't feel strong if you want to go with that. I have not yet encountered something else then crossbuild artifacts that we need to keep in special dir

Reply:
I am gonna put RPMs there 2 because rpmbuild isn't able to produce an rpm to current dir and always put the arch subfolder there, so _output it is